src/mono/wasm/runtime/types/blazor.ts

```diff
@@ -21,6 +21,8 @@ export interface BootJsonData {
     // These properties are tacked on, and not found in the boot.json file
     modifiableAssemblies: string | null;
     aspnetCoreBrowserTools: string | null;
+
+    readonly extensions?: { [name: string]: any };
```

Comment:
Is there specific use-case ?

Reply:
Yes, Blazor already has such extension, although it's empty at the moment. It can be configured using MSBuild `@(WasmBootConfigExtension)`

Reply:
How does the user code access that after we load and process it  ?